README.md

```diff
@@ -3,128 +3,95 @@
 [![CircleCI](https://circleci.com/gh/tulibraries/funcake_dags.svg?style=svg)](https://circleci.com/gh/tulibraries/funcake_dags)
 ![pylint Score](https://mperlet.github.io/pybadge/badges/9.47.svg)
 
-This is the repository for Funnel Cake (PA Digital / DPLA Data QA Interface) Airflow DAGs (Directed Acyclic Graphs, e.g., data processing workflows) for data indexing to Solr + related jobs. These DAGs are expecting to be run within an Airflow installation akin to the one built by our [TUL Airflow Playbook (private repository)](https://github.com/tulibraries/ansible-playbook-airflow).
+This is the repository for Funnel Cake (PA Digital / DPLA Data QA Interface) Airflow DAGs (Directed Acyclic Graphs, e.g., data processing workflows) for data indexing to Solr and related jobs. These DAGs are expecting to be run within an Airflow installation akin to the one built by our [TUL Airflow Playbook (private repository)](https://github.com/tulibraries/ansible-playbook-airflow).
 
-## Repository Structure
 
-This repository has 3 main groups of files:
-- Airflow DAG definition python files (ending with `_dag.py`);
-- Airflow DAG tasks python files used by the above (starting with `task_`);
-- and required local development, test, deployment, and CI files (`tests`, `configs`, `.travis`, Pipfile, etc.).
-
-## Airflow Expectations
-
-These the Airflow expectations for these Funnel Cake DAGs to successfully run:
+## Prequisites
 
 **Libraries & Packages**
 
-- Python 3.7
-- Python Packages: see the [Pipfile](Pipfile)
+- Python: Version as specified in `.python-version`, currently 3.6.8.
+- Pip: 18.1+
+- Pipenv: 2020-11-15+
+- Python Package Dependencies: see the [Pipfile](Pipfile)
+- Docker 20.10+
+- Docker-Compose: 1.27+
 
 **Airflow Variables**
 
+These variable are initially set in the `variables.json` file.
+
 - `FUNCAKE_CONFIGSET`: The SolrCloud Configset identifier to use for creating new Funnel Cake Collections & updating Aliases. Based on the https://github.com/tulibraries/funcake-solr latest release
 - `FUNCAKE_OAI_ENDPT`: The OAI Endpoint to be harvested for indexing.
 - `FUNCAKE_OAI_SET`: The OAI set used for harvesting from the OAI Endpoint (above). If all sets wanted, set to "" (but you have to set).
 - `FUNCAKE_MD_PREFIX`: The OAI metadata prefix used for harvesting from the OAI Endpoint (above). This is required, per OAI-PMH specifications.
 - `AIRFLOW_DATA_BUCKET`: The AWS S3 Bucket label (label / name, not ARN or URI) the harvested OAI-PMH XML data is put into / indexed from.
 - `AIRFLOW_HOME`: The Airflow system home directory path. Used here for locating our `scripts` repository when running the indexing bash script.
 
-See the rest of the expected variables in `variables.json`.
+Other expected variables are listed in [variables.json](variables.json)
 
 **Airflow Connections**
```

Comment:
If we are listing connections, we should add AIRFLOW_CONN_SLACK_WEBHOOK for slack failure notifications and FUNCAKE_SLACK_WEBHOOK for successes.